app/policies/follow_policy.rb

```diff
@@ -1,15 +1,7 @@
 class FollowPolicy < ApplicationPolicy
-  def update?
-    false
-  end
-
   def create?
     record.follower == user
   end
+  alias_method :update?, :create?
   alias_method :destroy?, :create?
-
-  def visible_attributes(all)
-    is_owner? ? all : all - %i[hidden]
-  end
-  alias_method :editable_attributes, :visible_attributes
```

Comment:
It was failing because follow doesn't have a `user` so `is_owner` doesn't work. trying to do something like `record.follower == user` gave `no such method follower`